index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sorting-Algorithms-in-Javascript</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/css/main.css">
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="navbar__container wrapper">
+            <div class="navbar__logo"><a href="#" class="navbar__logo-name">sora-al.js <img src="/img/flat-color-icons_generic-sorting-asc.png" alt="" srcset=""> </a> </div>
+            <div class="navbar__list-container">
+                <ul class="navbar__list">
+                    <li class="navbar__list-item"><a href="#news.html" class="navbar__list-link">News</a></li>
+                    <li class="navbar__list-item"><a href="#algorithm.html" class="navbar__list-link">Algorithm</a>
+                        <ul class="navbar__dropdown">
+                            <li class="navbar__dropdown-item"><a href="" class="navbar__dropdown-link">Bubble
```

Comment:
Kiepskie formatowanie - rozbij proszę elementy listy na kilka linijek, bo źle się to czyta. ;)